.github/workflows/ci.yml

```diff
@@ -74,10 +74,57 @@ jobs:
           DATABASE_URL: postgres://${{ env.PGUSER }}:${{ env.PGUSER }}@${{ env.PGHOST }}:${{ job.services.postgres.ports[5432] }}/${{ env.PGDATABASE }}?sslmode=require
           AWS_SKIP_CREDENTIALS: 1
           AWS_REGION: eu-central-1
+  musl-build:
+    name: Build ${{ matrix.target }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - target: aarch64-unknown-linux-musl
+          - target: x86_64-unknown-linux-musl
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+        with:
+          set-safe-directory: false
+      - uses: taiki-e/install-action@v2
+        with: { tool: cross }
+      - uses: Swatinem/rust-cache@v2
+        if: github.event_name != 'release' && github.event_name != 'workflow_dispatch'
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+        # https://github.com/docker/setup-qemu-action
+        with:
+          platforms: linux/amd64,linux/arm64
+      - name: Set up AWS SAM
+        uses: aws-actions/setup-sam@v2
+        with:
+          use-installer: true
```

Comment:
I don't quite get why the `sam`-CLI was installed in the docker build in the first place.

I think this is not nessesary, right?


```suggestion
```